helm/linkerd2-app/values.yaml

```diff
@@ -4,51 +4,95 @@
 
 # Values that are passed along to sub-charts
 global:
+  # The kubernetes cluster domain.
   clusterDomain: &cluster_domain cluster.local
+
+  # The cluster networks for which service discovery is performed. This should
+  # include the pod network but need not include the node network.
+  #
+  # By default, all private networks are specified so that resolution works in
+  # typical Kubernetes environments.
+  clusterNetworks: "10.0.0.0/8,100.64.0.0/10,172.16.0.0/12,192.168.0.0/16"
+
   imagePullPolicy: &image_pull_policy IfNotPresent
+  controllerLogLevel: &controller_log_level info
 
   # control plane trace configuration
   controlPlaneTracing: false
 
   # control plane version. See Proxy section for proxy version
-  linkerdVersion: &linkerd_version stable-2.8.1
-
-  identityTrustAnchorsPEM: |
-
-  identityTrustDomain: *cluster_domain
-
-  # GS changes: needed now as global value to be available in subcharts including _proxy.tpl
+  linkerdVersion: &linkerd_version stable-2.9.0
+
+  # GS changes: currently this value isn't used - the namespace
+  # is taken from the .Release.Namespace value.
+  namespace: linkerd
+
+  # enables the use of EndpointSlice informers for the destination service;
+  # enableEndpointSlices should be set to true only if EndpointSlice K8s feature gate is on;
+  # the feature is still experimental.
+  enableEndpointSlices: false
+
+  # enabling this omits the NET_ADMIN capability in the PSP
+  # and the proxy-init container when injecting the proxy;
+  # requires the linkerd-cni plugin to already be installed
+  cniEnabled: false
```

Comment:
still defaulting this to false, although i would always recommend using the CNI to customers. once we have the ability to have a parent > child relationship between Apps we can switch this on by default.